client/control/components/action_forms/HeartbeatForm.js

```diff
@@ -18,6 +19,11 @@ export default function HeartbeatForm({ fields }) {
         This action shows a single message or survey prompt to the user.
       </p>
       <div className="fluid-4">
+        <FormField
+          type="checkbox"
+          label="Include unique user ID in post-answer-url and Telemetry"
+          field={fields.includeTelemetryUUID}
+        />
```

Comment:
A minor thing, but this could probably look better visually. Do you know if that would be easy to improve? I think especially putting the checkbox and the label on the same line would nicer?

![shot_7335](https://cloud.githubusercontent.com/assets/305049/20191567/e9e2cdbe-a73a-11e6-9810-2f9f8d037a58.png)